crates/unleash-edge-feature-refresh/src/lib.rs

```diff
@@ -796,4 +824,530 @@ mod tests {
         assert!(tokens_to_refresh.contains(&etag_but_last_refreshed_ten_seconds_ago));
         assert!(tokens_to_refresh.contains(&no_etag_so_is_due_for_refresh));
     }
+
+    async fn client_api_test_server(
```

Comment:
All of these tests are just harvested from the integration tests. They're doing basically the same patterns and the only reason I can see that there were broken out was to reduce the LOC here